roles/kotlin/molecule/default/molecule.yml

```diff
@@ -3,7 +3,7 @@ driver:
   name: docker
 platforms:
   - name: instance
-    image: "geerlingguy/docker-ubuntu2004-ansible:latest"
+    image: "geerlingguy/docker-ubuntu2204-ansible:latest"
```

Comment:
I don't think we should change this to 22.04 since we are using wsl in the build of teamcity agent image which is pegged at 20.04 due to the jetbrains base image.

Reply:
Unintended change - reverting..